### ISSUE TITLE
assert custom message max string length

### ISSUE THREAD

**Issue by makcedward** [cucumber-rs/cucumber#332]
Recently, I have been trying to debug a failure case with a custom message. I noticed that it cannot print out all the messages if the string value is more than around 1000 (or 1024) characters.

I tested `assert!` without cucumber, and it was able to print out more than 1000 (or 1024) characters. I am curious about whether there are some limitations or additional configurations that could be affecting the printing of longer strings.

```
let msg = "abcdefghij";
assert!(false, "{}", msg.repeat(110));
```

**Comment by ilslv** [comment]
Thanks for reporting this! I've managed to pin down the issue and fix it https://github.com/cucumber-rs/cucumber/pull/333

**Comment by tyranron** [comment]
@makcedward fixed in [0.21.0](https://github.com/cucumber-rs/cucumber/releases/tag/v0.21.0).
